src/cli/device-protection.js

```diff
@@ -0,0 +1,59 @@
+module.exports = ({ commandProcessor, root }) => {
+	const deviceProtection = commandProcessor.createCategory(root, 'device-protection', 'Commands for managing device protection');
+
+	commandProcessor.createCommand(deviceProtection, 'status', 'Gets the current device protection status', {
+		handler: () => {
+			const DeviceProtectionCommands = require('../cmd/device-protection');
+			return new DeviceProtectionCommands().getStatus();
+		},
+		examples: {
+			'$0 $command': 'Gets the current device protection status'
+		}
+	});
+
+	commandProcessor.createCommand(deviceProtection, 'disable', 'Disables device protection', {
+		options: {
+			'open': {
+				boolean: true,
+				description: 'Turns a protected device an unprotected/open device'
+			}
+		},
+		handler: (args) => {
+			const DeviceProtectionCommands = require('../cmd/device-protection');
+			return new DeviceProtectionCommands(args).disableProtection(args);
+		},
+		examples: {
+			'$0 $command': 'Turns a protected device to being protected in service mode',
+			'$0 $command --open': 'Turns a protected device back to an unprotected/open device'
+		}
+	});
+
+	commandProcessor.createCommand(deviceProtection, 'enable', 'Enables device protection', {
+		options: {
+			file: {
+				description: 'Provide file to use for device protection'
+			}
+		},
+		handler: (args) => {
+			const DeviceProtectionCommands = require('../cmd/device-protection');
+			return new DeviceProtectionCommands().enableProtection(args);
+		},
+		examples: {
+			'$0 $command': 'Enables device protection'
+		}
+	});
+
+	commandProcessor.createCommand(deviceProtection, 'protect', 'Adds device-protection to your bootloader binary', {
```

Comment:
Let's move this to `particle binary enable-device-protection`
It will be confusing to have it next to enable, disable and status.